src/map-interactions.react.js

```diff
@@ -176,6 +186,9 @@ export default class MapInteractions extends Component {
     const pos = this._getTouchPos(event);
     this.setState({pos});
     this.props.onTouchEnd({pos});
+    if (!this.state.dragged) {
```

Comment:
We're not monitoring onMouseLeave, so there's a chance onMouseUp doesn't get hit when the mouse button is released. It's safer to reset it onMouseDown. Unless we want to do it in both places for clarity.